MDS_3/doc/MDS_3/PackageDescription.txt

```diff
@@ -0,0 +1,75 @@
+/// \defgroup PkgMDS3Ref 3D Mesh Data Structures Reference
+
+/// \defgroup PkgMDS3Concepts Concepts
+/// \ingroup PkgMDS3Ref
+/// The main concepts of this package.
+
+/// \defgroup PkgMDS3Classes Classes
+/// \ingroup PkgMDS3Ref
+/// The class in this group is the main class involved .
+
+/// \defgroup PkgMDS3Functions Functions
+/// \ingroup PkgMDS3Ref
+/// The functions to work with the 3D mesh data structures.
+
+/// \defgroup PkgMDS3ExportFunctions Export Functions
+/// \ingroup PkgMDS3Ref
+/// The free functions that can be used to export meshes to given output file formats.
+
+/// \defgroup PkgMDS3IOFunctions Input/Output Functions
+/// \ingroup PkgMDS3Ref
+/// The free functions that can be used to read and write meshes.
+
+/*!
+\addtogroup PkgMDS3Ref
+\todo check generated documentation
```

Comment:
this is a warning that we put when the doc move to doxygen, not needed for new package.